### PR TITLE
Document the new application interface

### PR DESCRIPTION
**Pull request by flipback**
Related on #65.

### CODE REVIEW

File: docs/source/1_getting_started/hello_world.rst [hunk 1]
```diff
@@ -100,11 +95,11 @@ Now we must include our node set to the information model of the application in
...
-However we do everything ourselves, so that we can learn some basic OPC UA conceptions. If you are already familiar to the protocol, just skip the rest of the section. 
+This might seem quite complicated, but actually you won't need to make your XML node sets manually. You can use our `OPC UA Designer`_ to make this process easier.
+However we do everything ourselves, so that we can learn some basic OPC UA conceptions. If you are already familiar to the protocol, just skip the rest of the section.
 
-*OPC UA Information* model is split into *namespaces*. Each of them must have its *namespace index*. Index 0 is reserved for the OPC UA standard namespace, where all standard types are described. 
-In our application it is stored in **Opc.Ua.NodeSet.xml**. The application can't work without it, so we need to include this file in our configuration. 
+*OPC UA Information* model is split into *namespaces*. Each of them must have its *namespace index*. Index 0 is reserved for the OPC UA standard namespace, where all standard types are described.
```
Comment (huebl) [inline docs/source/1_getting_started/hello_world.rst]: OPC UA defines two namespaces. The namespace with the namspace index 0 is the standard namespace. The namespace with the namespace index 1 is the namespace of the opc ua application server. These two namespaces must not be used by the application.

The OPC UA server currently does not offer the namespace 1 yet. This is a bug.

File: docs/source/1_getting_started/hello_world.rst [hunk 2]
```diff
@@ -140,7 +135,7 @@ The next important OPC UA conception is *references*, they describe relationship
 
     <Reference ReferenceType="Organizes" IsForward="false">i=85</Reference>
 
-The node ID 'i=85' references to standard node *Objects* in namespace 0 with identifier 85. You can found it in **Opc.Ua.NodeSet.xml**. 
+The node ID 'i=85' references to standard node *Objects* in namespace 0 with identifier 85. You can found it in **Opc.Ua.NodeSet.xml**.
```
Comment (huebl) [inline docs/source/1_getting_started/hello_world.rst]: This Nodeset is out of date, We must replace the nodeset by version 2.

File: docs/source/1_getting_started/hello_world.rst [hunk 3]
```diff
@@ -293,9 +288,9 @@ The last step is to write new value "Hello, World!" into the string:
     ptr->setValueSync(dataValue);
 
 OPC UA variables contain not only values, but some additional information. The *status code* provides information about the quality of the data. If it is not **Success** the client
-can't trust the value of the variable. 
+can't trust the value of the variable.
```
Comment (huebl) [inline docs/source/1_getting_started/hello_world.rst]: Additional there is a Source-Timestamp and a Server-Timestamp.

File: docs/source/1_getting_started/hello_world.rst (hunk 1 above)
Comment (flipback) [reply docs/source/1_getting_started/hello_world.rst]: I see. When we fix the bug, I'll update the document.

File: docs/source/1_getting_started/hello_world.rst [hunk 4]
```diff
@@ -100,11 +95,11 @@ Now we must include our node set to the information model of the application in
     <NodeSetFile>@CONF_DIR@/HelloWorldNodeSet.xml</NodeSetFile>
   </InformationModel>
 
-This might seem quite complicated, but actually you won't need to make your XML node sets manually. You can use our `OPC UA Designer`_ to make this process easier. 
-However we do everything ourselves, so that we can learn some basic OPC UA conceptions. If you are already familiar to the protocol, just skip the rest of the section. 
+This might seem quite complicated, but actually you won't need to make your XML node sets manually. You can use our `OPC UA Designer`_ to make this process easier.
```
[the remark this answers is not in the document]
Comment (flipback) [reply docs/source/1_getting_started/hello_world.rst]: I know, but I'd like to promote our solutions.

File: docs/source/1_getting_started/hello_world.rst (hunk 3 above)
Comment (flipback) [reply docs/source/1_getting_started/hello_world.rst]: Yes, it is so. However, I'd like to write about it in **Data  Access** section.

**Pull request merged**